.github/workflows/scripts/benchmark.sh

```diff
@@ -1,7 +1,77 @@
 #!/bin/bash
 
-LD_LIBRARY_PATH=$(pwd) apps/openssl speed
+execute_benchmark() {
+    LD_LIBRARY_PATH=$(pwd) apps/openssl speed
+
+    for i in sm3 sm4 aes-128-gcm aes-192-gcm aes-256-gcm; do
+        LD_LIBRARY_PATH="$(pwd)" apps/openssl speed -evp $i
+    done
+}
+
+calculate_Pn_position() {
```

Comment:
I'd split benchmark execution, results processing and the actual visualization into separate scripts so, e.g. the visualization can be changea and/or reused.

Reply:
It is a good idea, however I will keep it for the next iteration.